apache/cloudstack
# Run diagnostics API UI integration for system VMs and VR

## Description

This is a UI integration of the runDiagnostics API command that helps Admins troubleshoot network issues on CloudStack hosted networks by executing network-utility commands (ping, traceroute, arping) remotely on system VMs. #2721 


FS: https://cwiki.apache.org/confluence/display/CLOUDSTACK/CloudStack+Remote+Diagnostics+API


## Types of changes


## GitHub Issue/PRs






## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/19572501/44911310-bb726900-ad26-11e8-89bc-6139c181a83c.png)

Output
![image](https://user-images.githubusercontent.com/19572501/45024012-3e056c00-b038-11e8-81bd-1eb4eff42ddb.png)


## How Has This Been Tested?
As admin, log on to the CloudStack UI;

- Navigate to Infrastructure > System VMs or Virtual Router

- Click on the Run Diagnostics button and fill in the form that pops up.

See screenshots





## Checklist:


Testing
